stitch/src/androidTest/java/com/mongodb/stitch/StitchClientServiceTest.kt

```diff
@@ -270,4 +270,27 @@ class StitchClientServiceTest {
         assertFalse(mlsStitchClient.isAuthenticated)
         assertEquals(mlsStitchClient.loggedInProviderType, "")
     }
+
+    // NOTE: This test works locally when logging in with an email identity not associated with a Stitch user, but with our
```

Comment:
[nit] We should add a TODO here, so IntelliJ catches it.

Reply:
Can we also add tickets to go back to this?